sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/implementation/directconnectivity/ReplicatedResourceClientTest.java

```diff
@@ -20,6 +23,8 @@
 import reactor.core.Exceptions;
 import reactor.core.publisher.Mono;
 
+import java.net.URI;
+import java.net.URISyntaxException;
```

Comment:
Nit: extra imports , if you send another iteration

Reply:
Fixed